src/sandstorm/control-socket.capnp

```diff
@@ -0,0 +1,75 @@
+# Sandstorm - Personal Cloud Sandbox
+# Copyright (c) 2020 Sandstorm Development Group, Inc. and contributors
+# All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+@0xd4c5feffbd79e908;
+
+$import "/capnp/c++.capnp".namespace("sandstorm");
+
+using Util = import "util.capnp";
+
+interface Controller {
+  # Bootstrap interface offered by sandstorm on the control socket.
```

Comment:
Could use a little more text explaining that this is a control protocol for the Sandstorm server itself.

Reply:
It does mention that right after this, though I've tweak the wording so hopefully it reads less ambiguously as "this is what this is for" rather than "these commands happen to use it for this."